lib/Mojo/Server/Daemon.pm

```diff
@@ -5,9 +5,9 @@ use Carp qw(croak);
 use Mojo::IOLoop;
 use Mojo::Transaction::WebSocket;
 use Mojo::URL;
-use Mojo::Util      qw(term_escape);
+use Mojo::Util qw(term_escape);
```

Comment:
Make sure your perltidy is up to date, you seem to have untidied these lines.

Reply:
I wondered about that!  Thanks, I updated my Perl::Tidy from 20210717 to 20221112, re-tidied, and force-pushed.